site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -11,10 +10,11 @@ import { CreateWorkspacePageView } from "./CreateWorkspacePageView"
 const CreateWorkspacePage: FC = () => {
   const organizationId = useOrganizationId()
   const [searchParams] = useSearchParams()
-  const preSelectedTemplateName = searchParams.get("template")
+  const templateParam = searchParams.get("template")
+  const templateName = templateParam ? templateParam : ""
   const navigate = useNavigate()
   const [createWorkspaceState, send] = useMachine(createWorkspaceMachine, {
```

Comment:
I think this would ideally be a URL param like: "/templates/<name>/workspace"

Reply:
done in https://github.com/coder/coder/pull/2142/commits/2a191bad73ac902870f3b965ccafdd3eb309f970